### PR TITLE
TStips

### PR DESCRIPTION
**Pull request by OrchardPerformance**
More descriptions added and a few minor mistakes corrected.

### CODE REVIEW

File: firmware/integration/rusefi_config.txt [hunk 1]
```diff
@@ -858,7 +858,7 @@ custom pin_mode_e 1 bits,    U08,   @OFFSET@, [0:6], @@pin_mode_e_enum@@
 	float[CLT_CURVE_SIZE] cltIdleRpmBins;CLT-based target RPM for automatic idle controller;"C",        1,     0,   -100.0,    250.0,  2
 	float[CLT_CURVE_SIZE] cltIdleRpm;See idleRpmPid;"",      1,     0,      0.0,    8000.0,   0
 	
-	float targetVBatt;set targetvbatt VOLTS;"Volts",      1,      0,       0,30,      1
+	float targetVBatt;+This it the target battery voltage the alternator PID control will attempt to maintain;set targetvbatt VOLTS;"Volts",      1,      0,       0,30,      1
```
Comment (andreika-git) [inline firmware/integration/rusefi_config.txt]: "This **is** the target..." ?

File: firmware/integration/rusefi_config.txt [hunk 2]
```diff
@@ -871,8 +871,8 @@ custom pin_mode_e 1 bits,    U08,   @OFFSET@, [0:6], @@pin_mode_e_enum@@
 	brain_pin_e LIS302DLCsPin;
 
 
-	int tpsAccelLength;+How many engine cycles to consider?;"cycles",        1,     0,    1, 200,     0
-	float tpsAccelEnrichmentThreshold;+Maximum delta in TPS percentage over the 'length'. Actual TPS change has to be above this value in order to TPS/TPS acceleration to kick in.;"roc",        1,     0,    0, 200,     3
+	int tpsAccelLength;+This is the number of engine cycles that the TPS position chance can occur over, a longer duration will make the enrichment more active but too long may affect steady state driving, a good default is 30-60 cycles. ;"cycles",        1,     0,    1, 200,     0
```
Comment (andreika-git) [inline firmware/integration/rusefi_config.txt]: "chance" or "change"?

File: firmware/integration/rusefi_config.txt (hunk 1 above)
Comment (mck1117) [reply firmware/integration/rusefi_config.txt]: Sounds like proper English to me...

File: firmware/integration/rusefi_config.txt (hunk 2 above)
Comment (OrchardPerformance) [reply firmware/integration/rusefi_config.txt]: Looks like a spell checker misclick to be me, will correct.

File: firmware/integration/rusefi_config.txt (hunk 1 above)
Comment (OrchardPerformance) [reply firmware/integration/rusefi_config.txt]: Good spot, thanks for casting a second set of eyes over this as that kind of typo is not something a spell checker will pick up. Corrected and will do another PR.

**Pull request merged**